client/engine/chainservice/chainservice.go

```diff
@@ -0,0 +1,22 @@
+// Package chainservice is a chain service responsible for submitting blockchain transactions and relaying blockchain events
+package chainservice // import "github.com/statechannels/go-nitro/client/chainservice"
+
+import (
+	"math/big"
+
+	"github.com/statechannels/go-nitro/protocols"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// ChainEvent is an internal representation of a blockchain event
+type Event struct {
```

Comment:
As I understand it, this overly generic name may only be referred to in other packages as `chainservice.Event`, which is fine.